Mon Mar 26 23:18:23 FET 2012  Sergei Trofimovich <slyfox@community.haskell.org>
  * Fix List.foldl / Map.foldl ambiguosity
  
  Fixes the following error:
  src/runtime/haskell/PGF/Expr.hs:111:14:
      Ambiguous occurrence `foldl'
      It could refer to either `List.foldl',
                               imported from `Data.List' at src/runtime/haskell/PGF/Expr.hs:27:1-24
                               (and originally defined in `GHC.List')
                            or `Map.foldl',
                               imported from `Data.Map' at src/runtime/haskell/PGF/Expr.hs:28:1-40
diff -rN -u old-GF/src/runtime/haskell/PGF/Expr.hs new-GF/src/runtime/haskell/PGF/Expr.hs
--- old-GF/src/runtime/haskell/PGF/Expr.hs	2012-03-27 20:08:18.288228792 +0300
+++ new-GF/src/runtime/haskell/PGF/Expr.hs	2012-03-27 20:08:31.804059820 +0300
@@ -25,7 +25,7 @@
 import Data.Char
 import Data.Maybe
 import Data.List as List
-import Data.Map as Map hiding (showTree)
+import qualified Data.Map as Map hiding (showTree)
 import Control.Monad
 import qualified Text.PrettyPrint as PP
 import qualified Text.ParserCombinators.ReadP as RP
